### PR TITLE
ImShowPlotGenerator

### PR DESCRIPTION
**Pull request by galwiner**
Fixed issue #155

### CODE REVIEW

File: entropylab/api/plot.py [hunk 1]
```diff
@@ -99,8 +99,11 @@ def __init__(self) -> None:
         super().__init__()
 
     def plot_plotly(self, figure: go.Figure, data, **kwargs) -> None:
-        # TODO: Validate type of data
-        return px.imshow(data)
+        fig = px.imshow(data)
+        figure.add_trace(fig.data[0])
+        #todo add kwargs for setting x,y, c axes
```
Comment (urig) [inline entropylab/api/plot.py:104]: May I suggest implementing the #todo before merging the PR? :)

File: entropylab/api/plot.py [hunk 2]
```diff
@@ -99,8 +99,11 @@ def __init__(self) -> None:
         super().__init__()
 
     def plot_plotly(self, figure: go.Figure, data, **kwargs) -> None:
-        # TODO: Validate type of data
-        return px.imshow(data)
+        fig = px.imshow(data)
```
Comment (urig) [inline entropylab/api/plot.py:102]: `fig` and `figure` are a bit confusing. Is `fig` of type `Figure`? Maybe rename `fig` to `imshow_figure`?

**Review by urig — COMMENTED** [review]
I think a unit test for this is important.

**Comment by galwiner** [comment]
I'm merging this as @urig and I already went over it

**Pull request merged**